### DISCUSSION TITLE
Closed submission deleted or purged ?

### DISCUSSION THREAD

**Discussion by zazzati** [globaleaks/globaleaks-whistleblowing-software#4542] [Q&A]
**Current behavior**
We have a client that uses an instance of globakleaks
and who reported to us about the disappearance of the submissions in a "closed" state from the list available to the recipient.

The removal time of the submissions is set to 0, so they should never be removed.
There are other parameters that can allow the cancellation of the submissions closed after
a certain period of time?

Ve have not proceeded to update the application during the period of use by the customer

**Expected behavior**

permanently present submissions

**GlobaLeaks version:**
GlobaLeaks version: 3.9.15
Database version: 49

**Server Operating System and Version (if applicable):**
Ubuntu bionic. 

tnhx

**Comment by evilaliv3** [comment]
Dear @zazzati, actually submissions are deleted only in case of one of the following conditions:
- in case the submission expires in relation to the data retention policy configured onto the platform
- in case the submission is manually deleted by a recipient

Is it possible that when your client has received the submission your system was still configured with a data retention policy of 90 days and only after this period you changed it to 0 (to disable the data retention policy)?

**Comment by zazzati** [comment]
@evilaliv3 

retention policy actually set to 0 (it should disable automatic submission deletion).

thx
